.github/workflows/ci.yml

```diff
@@ -108,13 +107,15 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         uses: FreeRTOS/CI-CD-Github-Actions/link-verifier@main
+        with:
+          allowlist-file: ./.github/linkAllowList.txt
```

Comment:
Is the point of this to only allow a single, junk Url? Why not just allow the default scans to run?

Reply:
It was used in the unit tests, I could update it to be valid link but the action was failing because of this url